translations/messages.nl.php

```diff
@@ -24,18 +24,18 @@
         'cancel' => 'Annuleren',
         'confirm' => 'Bevestigen',
         'home' => 'Naar de homepagina',
-        'organisation_noun' => 'organisatie',
-        'suite_name' => 'OpenConext',
+        'organisation_noun' => 'bibliotheek',
+        'suite_name' => 'BiebConext',
```

Comment:
Should this be actually committed?

Reply:
Lolz, that was my test setup. Will revert that! Thanks for spotting that